fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FCU/A320_Neo_FCU.js

```diff
@@ -830,10 +830,13 @@ class A320_Neo_FCU_Heading extends A320_Neo_FCU_Component {
     }
 
     onTRKModeChanged(_newValue) {
-        if (_newValue) {
-            this.selectedValue = this.calculateTrackForHeading(this.selectedValue);
-        } else {
-            this.selectedValue = this.calculateHeadingForTrack(this.selectedValue);
+        // do not alter if we're in managed heading
```

Comment:
What if a heading is pre-selected and you swap the ref mode?

Reply:
Yeah this might not work.. used the same approach now as the PFD by using the dedicated simvar. Probably not how the real thing works but at least it's aligned now.

Reply:
I don't know if this exactly what was meant here, but the way the real thing works is that it sets the current selected hdg to the current actual trk, if the actual hdg was was within 5° of the selected hdg. If not, the value will stay the same. This applies no matter if a managed or a selected mode was engaged, as long as there is a value displayed and not dashes

Reply:
Yeah I have reverted the change, not sure if the behaviour you described is implemented in the FCU currently, but that would rather be part of another PR (or even better, the new FCU eventually)

Reply:
No, the current FCU just magically knows the wind and always just calculates the corresponding HDG/TRK. The new FCU does it like I described